app/assets/images/prx_auth-rails/user.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I guess someday (for the "global nav" project), we'd want to move all assets/styles into ID itself.  But it is not this day!